plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -105,6 +106,31 @@ export default function transformProps(
   const series: SeriesOption[] = [];
   const formatter = getNumberFormatter(contributionMode ? ',.0%' : yAxisFormat);
 
+  const totalStackedValues: number[] = [];
+  const showValueIndexes: number[] = [];
+
+  if (stack) {
+    rebasedData.forEach(data => {
+      const values = Object.keys(data).reduce((prev, curr) => {
+        if (curr === '__timestamp') {
+          return prev;
+        }
+        const value = data[curr] || 0;
+        return prev + (value as number);
+      }, 0);
+      totalStackedValues.push(values);
+    });
+
+    rawSeries.forEach((entry, seriesIndex) => {
+      const { data = [] } = entry;
+      (data as [Date, number][]).forEach((datum, dataIndex) => {
+        if (datum[1] !== null) {
+          showValueIndexes[dataIndex] = seriesIndex;
```

Comment:
`showValueIndexes` records the highest series index (is not a null value) of each data, so that we can show the total value at the top.